### ISSUE TITLE
Some questions about the demo (infering on single images in general)

### ISSUE THREAD

**Issue by gozi1123** [facebookresearch/DistDepth#20]
What is the 1.312 constant in output_to_depth function?
And what are the 0.1 and 10 inputs to the function? is it based on the training data and should change during inference? 
In the demo visualization you use 0.1 and 5 instead of 0.1 and 10
Is the output of output_to_depth in meters? and output of the net (the sigmoid) is disparity?
I didn't see any reference in the repo or the paper...
Thanks!

**Comment by choyingw** [comment]
The inference follows Monodepth2’s style (see the script https://github.com/nianticlabs/monodepth2/blob/master/evaluate_depth.py). The network output is in inverse depth (disparity) and to convert to meters, it takes min and max, here 0 mand 10m when training, from simsin’s statistics and convert to meters based on min and max.

1.312 is the stereo factor in simsin w.r.t. baseline (13.12cm, 0.1 as a unit here, also see the mentioned script). 

We use 0, 10 as min and max in disp-_to_depth function, but to visualize most depth ranges are in [0,5]m. Using [0,10]m you’ll see most areas are bluish and indistinguishable.
